hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/ssl/ReloadingX509KeyManager.java

```diff
@@ -64,7 +65,7 @@ public class ReloadingX509KeyManager extends X509ExtendedKeyManager {
    * materials are changed.
    */
   private PrivateKey currentPrivateKey;
-  private String currentCertId;
+  private List<String> currentCertList = new ArrayList<>();
```

Comment:
Thanks @ChenSammi for the patch with changes to the keystore and truststore.
_Minor_: may read better if `currentCertList` _renamed_ to `currentCertIdsList` or similar to differentiate and emphasize list of serialIds from list of certificate objects.